worker/buildbot_worker/test/util/misc.py

```diff
@@ -23,14 +23,16 @@
 except ImportError:
     # Python 3
     import builtins
-import cStringIO
 import errno
 import os
 import re
 import shutil
 import sys
+from io import BytesIO
+from io import StringIO
 
 import mock
```

Comment:
Well, `mock` offers the functionality that is:

* not related to future (=> an empty line)
* more specific to the content of this file (=> move after imports that are more generic)